itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -1504,12 +1504,20 @@ public void testDumpExternalTableSetFalse() throws Throwable {
             .run("use " + replicatedDbName)
             .run("show tables like 't3'")
             .verifyResult("t3")
+            .verifyExternalTable(replicatedDbName, "t3",
+                    primary.getTableLocation(primaryDbName, "t3"))
             .run("select id from t3 where id = 10")
-            .verifyFailure(new String[] {"10"});
+            // Since both the external tables point to the same location as they share the same
```

Comment:
Why don't we create separate DFS cluster for primary and replica? It helps to have different file system for source and target.

Reply:
The test is designed to have primary and replica on the same DFS. May be we want to add another test for having primary and replica on two different DFSes. I will anyway test that scenario.